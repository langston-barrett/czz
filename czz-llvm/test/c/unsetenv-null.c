#include <errno.h>
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
  printf("return=%d\n", unsetenv(NULL));
  printf("errno=%d\n", errno);
  return 0;
}
