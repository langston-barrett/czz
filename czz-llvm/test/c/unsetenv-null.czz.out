return=-1
errno=22
